model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,91 @@
+groups:
+  - id: metric.telemetry.sdk.span.ended_count
```

Comment:
I was thinking about started spans but considered it less useful because it wouldn't allow the computation of the inflight spans:

Two my knowledge the absolute value for counters is irrelevant and not really queryable for most metric backends: You only are able to compute the increase of a counter between two points in time.
And you can't compute the inflight spans using `increase(ended)-increase(started)`. The same also applies to when using `DELTA` temporality.

However, I did not consider adding an `inflight` updown-counter directly. That would also allow the computation of started spans via `increase(ended)+last_value(inflight)`. So I'm definitiely in favor of adding this metric.

However, I'm not sure whether `inflight` is the best name here, as for me inflight intuitively sounds like "being sent over the wire" when looking at the full telemetry system from the outside. Maybe something like `active` or `not_ended`? I'm happy with other suggestion or even `inflight`, just want to ensure that we are truly happy with the name here